cves/CVE-2014-0050.yml

```diff
@@ -118,7 +130,9 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
+  answer: |
+    Searched through each link on cve.mitre.org as well as a thorough search of the git log and found 
+    nothing on the origin of this bug. It was initially reported by Peter Magnusson.
```

Comment:
Start with "the bug was initially reported by Peter Magnusson" and try to consolidate the rest, eg. "I was unable to find any further information."